index.js

```diff
@@ -15,17 +15,9 @@ const mysqlOptions = process.env.CLEARDB_DATABASE_URL ? process.env.CLEARDB_DATA
     database: process.env.DB_NAME,
 }
 
-console.log(mysqlOptions)
-
-const conn = mysql.createConnection(mysqlOptions)
-
-conn.connect((err) => {
-    if (err) {
-        console.log('ERR: Could not connect to DB. ' + err)
-        return
-    }
```

Comment:
Why has this been removed? Is there no chance of an error occurring when you use a pool to connect rather than a single connection?

Reply:
So I looked into this. I _think_ if a single connection fails, the pool will try make a new one, so we shouldn't care about single connection errors.

We can however add this to the `start()` method. I think it might be a good idea.
```
const conn = mysql.createPool(mysqlOptions)
conn.on('error', (err) => console.log('ERR: Could not connect to DB. ' + err))
```